addons/l10n_account_edi_ubl_cii_tests/tests/test_files/from_odoo/facturx_out_invoice.xml

```diff
@@ -38,7 +38,7 @@
         </ram:NetPriceProductTradePrice>
       </ram:SpecifiedLineTradeAgreement>
       <ram:SpecifiedLineTradeDelivery>
-        <ram:BilledQuantity unitCode="DZN">2.0</ram:BilledQuantity>
+        <ram:BilledQuantity unitCode="KGM">2.0</ram:BilledQuantity>
```

Comment:
The test had a goal of testing something with UoMs and I am not sure that goal is still fulfilled this way.  The problem a bit with UBL, is that we do not define a field on the UoM to fill it, so we can not create Dozen in the test just like that.  

Even if we add a field in the UoM (which you can find back in a lot of localizations), the problem is that if you define a package of 12, you can not expect the user to go add DZN in there.  So, maybe the simplest thing is to convert the quantities to the one used in the main UoM of the product.  (and not let people add packaging uoms as base uom)